source/documentation/tools/upgrading.md

```diff
@@ -0,0 +1,38 @@
+# Upgrading RStudio
```

Comment:
Should this be under the R Studio section as it just seems to relate to R Studio?

Reply:
Yeah, I think long term the layout should look something like this:

- rstudio/index.html
- rstudio/upgrading.html
- jupyterlab/index.html
- jupyterlab/upgrading.html 


Mind if we leave it as it is for now and address this n a future PR?

Reply:
sure